.circleci/config.yml

```diff
@@ -0,0 +1,172 @@
+defaults: &defaults
+  working_directory: ~/circleci-champaign
+  docker:
+    - image: soutech/ruby2.4.1_node_awscli:0.2
+
+version: 2
+jobs:
+  build:
+    <<: *defaults
+    environment:
+      EXTERNAL_ASSETS_JS_PATH: /home/circleci/circleci-champaign/tmp/external_assets_source/javascripts/champaign
+      EXTERNAL_ASSETS_PATH: /home/circleci/circleci-champaign/tmp/external_assets_source/
+    steps:
+      - checkout
+      # Restore gem, JS and assets caches
+      - restore_cache:
+          key: gem-cache-{{ checksum "Gemfile.lock" }}
+      - restore_cache:
+          key: yarn-cache-{{ checksum "yarn.lock" }}
+      - restore_cache:
+          # change epoch to git commit sha of the last commit in the sou-assets repo (Branch is either current branch, $DEFAULT_ASSET_BRANCH, or 'master', in that order):
+          # git ls-remote git://github.com/SumOfUs/sou-assets.git \ grep refs/heads/master | cut -f 1
+          key: assets-cache-{{ epoch }}
```

Comment:
The assets cache key now uses time so there'll never be a cache hit. I left it like this to keep in mind that it's WIP, @osahyoun requested that we get this in as-is in order to benefit from it as soon as possible.